tests/rptest/services/admin.py

```diff
@@ -255,9 +255,11 @@ def is_leader_stable():
             info = self.wait_stable_configuration(topic, partition, namespace,
                                                   replication, timeout_s,
                                                   hosts)
-            return check(info.leader)
+            if check(info.leader):
+                return True, info.leader
+            return False
 
-        wait_until(
+        return wait_until_result(
```

Comment:
@rystsov why did this change? `wait_until_result` is intended for when the return value of the predicate is desired by the caller of wait_until, but here it is thrown away.

Reply:
Sorry! I didn't see the `return` :)